docker/kubernetes-tentacle/dev/Dockerfile

```
@@ -64,6 +65,7 @@ ENV OCTOPUS__K8STENTACLE__PODVOLUMEJSON=""
 ENV OCTOPUS__K8STENTACLE__DISABLEAUTOPODCLEANUP="False"
 ENV TentacleHome=""
 ENV TentacleApplications=""
+ENV TentacleCertificate=""
```

should we specify that this is Base64?
```suggestion
ENV TentacleCertificateBase64=""
```